components/tcp_transport/transport_ws.c

```diff
@@ -711,6 +713,11 @@ esp_err_t esp_transport_ws_set_config(esp_transport_handle_t t, const esp_transp
     return err;
 }
 
+bool esp_transport_ws_get_fin_flag(esp_transport_handle_t t) {
+  transport_ws_t *ws = esp_transport_get_context_data(t);
+  return ws->frame_state.fin;
```

Comment:
nitpick: IDF formatting (spaces, fn braces)
```suggestion
bool esp_transport_ws_get_fin_flag(esp_transport_handle_t t)
{
    transport_ws_t *ws = esp_transport_get_context_data(t);
    return ws->frame_state.fin;
```